### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Accessible environment_Desired Storage Explorer theme_Name Role Value: Incorrect Role is assigned for Documentation and Troubleshooting Guide as buttons.

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/AzureStorageExplorer#7605]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.32.1(96)

### Architecture

x64

### Storage Explorer Build Number

20231114.10

### Platform

macOS

### OS Version

MAC Version 13.6.1

### GitHub Tags:
#A11yMAS;#A11yTCS;#MAC;#DesktopApp;#A11ySev2;#BM_AzureToolsStorageExplorer_MAC_Dec2023;#Azure Tools Storage Explorer;#FTP;#Voiceover;#WCAG4.1.2;#Name Role Value;#AILimited;

### Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.32.1 (96)
Screen Reader: Voice Over

### Steps to Reproduce

1. Launch Storage Explorer
2. Tab Navigate to "Get started" Tab and click on it.
3. Tab Navigate to "Documentation and Troubleshooting Guide".
4. Turn on voice over and Observe that Incorrect Role is assigned for Documentation and Troubleshooting Guide as buttons.

### Actual Experience

Incorrect Role is assigned for Documentation and Troubleshooting Guide as buttons.

### Expected Experience

Documentation and Troubleshooting Guide should be defined as Links.

### User Impact:
Screen reader users will face difficulty if Incorrect Role is assigned for Documentation and Troubleshooting Guide as buttons.

### Attachment:
![A11y_Azure Tools Storage Explorer_Accessible environment_Desired Storage Explorer theme_Name Role Value](https://github.com/microsoft/AzureStorageExplorer/assets/98728920/81562151-bd88-497d-907d-8a8740371fe0)

**Comment by kupatkar99** [comment]
verified the issue in below environment:
Version: 1.33.0-dev (96)
Platform: macOS
Architecture: x64
Build Number: 20240111.7
issue is not fixed please find added attachment for reference.
<img width="1049" alt="image (4)" src="https://github.com/microsoft/AzureStorageExplorer/assets/98530069/e4d90405-efa9-4100-bb5f-bb450c56cff4">

**Comment by craxal** [comment]
@kupatkar99 This was fixed quite a while ago. Please verify and close.

![image](https://github.com/microsoft/AzureStorageExplorer/assets/3473026/369a7f83-2925-4f63-ada5-2007b592bbb8)

**Comment by kupatkar99** [comment]
Verified the issue in below environment:
Microsoft Azure Storage Explorer
Version: 1.33.0 (97)
AzCopy Version: 10.23.0
Platform: macOS
Architecture: arm64
Build Number: 20240220.6
Issue is fixed. hence closing the bug please find added attachment for reference.
![Uploading 7605_fixed.png…]()

**Comment by kupatkar99** [comment]
Verified the issue in below environment:
Microsoft Azure Storage Explorer
Version: 1.33.0 (97)
AzCopy Version: 10.23.0
Platform: macOS
Architecture: arm64
Build Number: 20240220.6
Issue is fixed. hence closing the bug please find added attachment for reference.
![Uploading 7605_fixed.png…]()
